ee/tabby-webserver/src/service/background_job/third_party_integration/pulls.rs

```diff
@@ -49,11 +50,28 @@ pub async fn list_github_pulls(
                 let url = pull.html_url.map(|url| url.to_string()).unwrap_or_else(|| pull.url);
                 let title = pull.title.clone().unwrap_or_default();
                 let body = pull.body.clone().unwrap_or_default();
+
+                let author = pull.user.as_ref().map(|user| user.login.clone()).unwrap_or_default();
+                let email = if !author.is_empty() {
+                    match octocrab.users(&author).profile().await {
+                        Ok(profile) => {
+                            profile.email.unwrap_or_default()
+                        }
```

Comment:
if email doesn't present, should use None to represent (rather than use `""`).

This essentially make`author_email` as an optional field of `IssueDocument` / `PullDocument`